src/test/kotlin/net/atos/zac/app/inboxDocumenten/InboxDocumentenServiceTest.kt

```diff
@@ -0,0 +1,71 @@
+/*
+ * SPDX-FileCopyrightText: 2025 INFO.nl
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+package net.atos.zac.app.inboxDocumenten
+
+import io.kotest.core.spec.style.BehaviorSpec
+import io.kotest.matchers.shouldBe
+import io.mockk.Runs
+import io.mockk.checkUnnecessaryStub
+import io.mockk.every
+import io.mockk.just
+import io.mockk.mockk
+import jakarta.persistence.EntityManager
+import net.atos.client.zgw.drc.DrcClientService
+import net.atos.zac.documenten.InboxDocumentenService
+import net.atos.zac.documenten.model.InboxDocument
+import nl.info.client.zgw.drc.model.generated.EnkelvoudigInformatieObject
+import nl.info.client.zgw.zrc.ZrcClientService
+import java.time.LocalDate
+import java.util.UUID
+
+class InboxDocumentenServiceTest : BehaviorSpec({
+    val entityManager = mockk<EntityManager>(relaxed = true)
+    val drcClientService = mockk<DrcClientService>()
+    val zrcClientService = mockk<ZrcClientService>()
+
+    val inboxDocumentenService = InboxDocumentenService(
+        zrcClientService,
+        drcClientService
+    )
+
+    val field = InboxDocumentenService::class.java.getDeclaredField("entityManager")
```

Comment:
So this is a pretty hash way of getting the entity manager injected into the service. I would look at adding it as a constructor parameter.